crates/storage/provider/src/post_state.rs

```diff
@@ -1,17 +1,30 @@
 //! Output of execution.
 use reth_db::{
     cursor::{DbCursorRO, DbCursorRW, DbDupCursorRO, DbDupCursorRW},
-    models::{AccountBeforeTx, TransitionIdAddress},
+    models::{AccountBeforeTx, BlockNumberAddress},
     tables,
     transaction::{DbTx, DbTxMut},
     Error as DbError,
 };
 use reth_primitives::{
-    bloom::logs_bloom, proofs::calculate_receipt_root_ref, Account, Address, Bloom, Bytecode, Log,
-    Receipt, StorageEntry, TransitionId, H256, U256,
+    bloom::logs_bloom, proofs::calculate_receipt_root_ref, Account, Address, BlockNumber, Bloom,
+    Bytecode, Log, Receipt, StorageEntry, H256, U256,
 };
 use std::collections::BTreeMap;
 
+/// Storage for an account with the old and new values for each slot.
```

Comment:
```suggestion
/// Storage for an account with the old and new values for each slot: (slot -> (old, new))
```

maybe consider helper struct for old,new pair, IIRC revm had something like this as well

Reply:
I think we will have one more iteration on `PostState` in the future, but for now to not introduce conflict, lets leave it as it is.